### DISCUSSION TITLE
Add Categories to ease navigation

### DISCUSSION THREAD

**Discussion by rossilor95** [codinasion/codinasion-programme#1341] [Ideas]
How about assigning categories to the snippets? Some examples:

- Simple Numbers (Find a digit in a specific place in a number, find generic root, etc)
- Calculation
- Geometry
- Special Numbers (Fibonacci, Palindrome, Armstrong, etc.)
- Unit Conversion

**Comment by harshraj8843** [comment]
👋🏻 Hey @rossilor95 

That will be awesome 👍🏻 

This feature will be added soon 🚀 

<details>
<summary>Contribute to this feature</summary>

- You are welcome to raise a PR for https://github.com/codinasion/codinasion-programme/issues/891
- Add category in `README.md` for specific programmes

```diff
...
image: hero.png
+ category:
+   - addition
+   - simple numbers
tags:
  - python
  - cs
  - cpp
  - java
  - go
  - c
  - js
contributors:
  - harshi1122
...
```

</details>
